egs/wsj/s5/utils/data/limit_feature_dim.sh

```diff
@@ -3,6 +3,9 @@
 # Copyright 2016  Alibaba Robotics Corp. (author: Xingyu Na)
 # Apache 2.0
 
+# This script allows you to create a data directory from limited
+# dim range of the source data dir.
+
```

Comment:
OK, but I'd prefer adding it also to the message that gets printed out when incorrect number of parameters.
Also, perhaps the wording might be more explanatory -- something like 

The script creates a new data directory by selecting/copying a specified dimension range of the features in the source directory. 
Or something to that extent..